src/main/java/com/moorhuhnservice/moorhuhnservice/data/mapper/QuestionMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.moorhuhnservice.moorhuhnservice.data.mapper;
+
+import com.moorhuhnservice.moorhuhnservice.data.Question;
+import com.moorhuhnservice.moorhuhnservice.data.QuestionDTO;
+import java.util.Set;
+import org.mapstruct.Mapper;
+
+@Mapper(componentModel = "spring")
+public interface QuestionMapper {
+  QuestionDTO questionToQuestionDTO(final Question Question);
+
+  Question questionDTOToQuestion(final QuestionDTO questionDTO);
+
+  Set<Question> questionDTOsToQuestions(final Set<QuestionDTO> questionDTOs);
+
+  Set<QuestionDTO> questionsToQuestionDTOs(final Set<Question> questions);
```

Comment:
Is there a reason why this can't be
```suggestion
  Collection<Question> questionDTOsToQuestions(final Collection<QuestionDTO> questionDTOs);

  Collection<QuestionDTO> questionsToQuestionDTOs(final Collection<Question> questions);
```

Reply:
This will break too much code: `incompatible types: java.util.Collection<com.moorhuhnservice.moorhuhnservice.data.Question> cannot be converted to java.util.Set<com.moorhuhnservice.moorhuhnservice.data.Question>`